jobserver/management/commands/create_backend.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+
+from jobserver.models.backend import Backend
+from jobserver.models.user import User
```

Comment:
Nit: We've generally opted to keep model imports in the form `from jobserver.models import MyModel` to avoid the specific location being relevant to the calling code.  We've moved models around a couple of times at this point so it's paid off there.  It's not _necessary_ but consistency is useful, etc.